test/test_torch.py

```diff
@@ -1837,6 +1837,25 @@ def _test_logical_not(self, device):
     def test_logical_not(self):
         self._test_logical_not(self, 'cpu')
 
+    @staticmethod
+    def _test_logical_xor(self, device):
+        for dtype in (torch.bool,):  # Will add more dtypes in the future
+            expected_res = torch.tensor([0, 0, 1, 1], dtype=dtype, device=device)
+            a = torch.tensor([10, 0, 1, 0], dtype=dtype, device=device)
+            b = torch.tensor([1, 0, 0, 10], dtype=dtype, device=device)
+            # new tensor
+            self.assertEqual(expected_res, a.logical_xor(b))
+            # out
+            c = torch.empty(0, dtype=dtype, device=device)
+            torch.logical_xor(a, b, out=c)
+            self.assertEqual(expected_res, c)
+            # in-place
+            a.logical_xor_(b)
+            self.assertEqual(expected_res, a)
```

Comment:
same as logical_not -- can we test `out=` with the wrong dtype?  (Also what happens?)